Specs/Widgets/Timeline/TimelineSpec.js

```diff
@@ -1,25 +1,159 @@
-import { Clock } from "../../../Source/Cesium.js";
-import { Timeline } from "../../../Source/Cesium.js";
+import {
+  TimelineHighlightRange,
+  Clock,
+  Timeline,
+  JulianDate,
+  Color,
+} from "../../../Source/Cesium.js";
 
+function getHighlightElements(root) {
+  return root.getElementsByClassName("cesium-timeline-highlight");
+}
 describe("Widgets/Timeline/Timeline", function () {
-  var container;
+  var container, timeline, timelineElement;
   beforeEach(function () {
     container = document.createElement("span");
     container.id = "container";
     container.style.width = "1px";
     container.style.height = "1px";
     document.body.appendChild(container);
+    timeline = new Timeline(container, new Clock());
+    timeline.render(); // TODO::resize will now use render under the hood "resize"
+    timelineElement = container.getElementsByClassName(
+      "cesium-timeline-main"
+    )[0];
+    // TODO::make _makeTics static and private
```

Comment:
How do you plan on addressing these TODOs?  We don't leave inline TODOs because they're easy to lose track of once the code is merged.  If it's something that should be fixed for this PR, we should figure out how to solve it, or if it's an unrelated problem we should open an issue and remove these inline TODO messages.

Reply:
I've implemented these TODO and deleted them